### DISCUSSION TITLE
Why RedisTagAwareAdapter do not accept any policy eviction?

### DISCUSSION THREAD

**Discussion by Cyanat** [symfony/symfony#51083] [Q&A] [answered]
Hello!
I see [here](https://github.com/symfony/symfony/blob/d0b8685b34bcd5f9f4421172f662cf87b7bf3a01/src/Symfony/Component/Cache/Adapter/RedisTagAwareAdapter.php#L87C9-L87C9) that RedisTagAwareAdapter do not accept policy eviction allkeys-lfu. What would be the reason for that?

Also, in [symfony doc](https://symfony.com/doc/current/components/cache/adapters/redis_adapter.html#configuring-redis),  it's recommanded to use `maxmemory-policy allkeys-lru`
which is in contradiction with the code of RedisTagAwareAdapter as far I understand...

**Comment by MatTheCat** [comment, marked as answer]
Hi, when using `RedisTagAwareAdapter` you’re not supposed to configure an `allkeys-*` eviction because you could lose relationships between tags and cache items.

I think the [Configuring Redis](https://symfony.com/doc/current/components/cache/adapters/redis_adapter.html#configuring-redis) section is simply misplaced: it should be before and at the same level than [Working with Tags](https://symfony.com/doc/current/components/cache/adapters/redis_adapter.html#working-with-tags). The latter should also warn about supported eviction policies. Do you feel like opening a PR? :grinning:

**Comment by Cyanat** [comment]
OK, thanks for your feedback, PR created.
